rpcd/playbooks/roles/rpc_maas/templates/rsyslogd_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
+label: "rsyslogd_process_check--{{ ansible_hostname }}"
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : process_check.py
+    args    : [{% for name in rsyslogd_process_names %}{{ name }}{% if not loop.last %}, {% endif %}{% endfor %}, {% if "{{ ansible_ssh_host }}" in groups["all_containers"] %} -c "{{ ansible_ssh_host }}" {% else %} {% endif %}]
+alarms      :
+{% for process in rsyslogd_process_names %}
+    {{ process }}_process_status:
+        label                   : {{ process }}_process_status--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["{{ process }}_process_status"] != 1 ) {
+                return new AlarmStatus(CRITICAL, "Rsyslogd process {{ process }} not running on {{ ansible_hostname }}.");
+            }
+{% endfor %}
```

Comment:
Has anyone addressed this? Is this a problem with the check

```
inventory_hostname in groups['all_containers']
```

Perhaps we don't want `inventory_hostname` there?